docs/reference.rst

```diff
@@ -465,6 +465,12 @@ Properties
 
 :ref:`Progress <gdbus-property-de-pengutronix-rauc-Installer.Progress>` readable   (isi)
 
+:ref:`Compatible <gdbus-property-de-pengutronix-rauc-Installer.Compatible>` readable   s
+
+:ref:`Variant <gdbus-property-de-pengutronix-rauc-Installer.Variant>` readable   s
+
+:ref:`Bootslot <gdbus-property-de-pengutronix-rauc-Installer.Bootslot>` readable   s
```

Comment:
Could we rename this to BootSlot for consistency with the existing properties?

Reply:
Changed to BootSlot